compiler warning

I noticed that Android ndk clang gave this compiler warning:
```
[ 98%] Building C object CMakeFiles/baresip.dir/src/vidutil.c.o
/usr/src/libbaresip-android/baresip/src/video.c:519:42: warning: overflow in expression; result is 1650065408 with type 'long' [-Winteger-overflow]
        const uint64_t max_delay = PKT_SIZE * 8 * 1000000L / bitrate + 1;
                                                ^
1 warning generated.
```
I'm not sure if it is correct because the const is of type uint64, not long.

--- a/src/video.c
+++ b/src/video.c
@@ -516,9 +516,9 @@ static int vtx_thread(void *arg)
 	else
 		bitrate = vtx->video->cfg.bitrate;
 
-	const uint64_t max_delay = PKT_SIZE * 8 * 1000000L / bitrate + 1;
+	const uint64_t max_delay = PKT_SIZE * 8 * 1000000LL / bitrate + 1;
 	const uint64_t max_burst =
-		vtx->video->cfg.burst_bits * 1000000L / bitrate;
+		vtx->video->cfg.burst_bits * 1000000LL / bitrate;
 
 	struct vidqent *qent = NULL;
 	struct mbuf *mbd;

